.ci/docker/common/install_pytorch.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
lots of duplication code in .ci/scripts/utils.sh

is it possible to dedupe?

Reply:
Yeah, I will follow up with another BE diff to dedup this. The underlying issue here is that Linux build uses those in `.ci/docker/common`, while MacOS build uses those under `.ci/scripts`.  There are subtle difference for example `as_ci_user`, so they are worth another PR